content/post/40-js-ipfs-0.30.md

```diff
@@ -0,0 +1,63 @@
+---
+date: 2018-07-09
+url: 40-js-ipfs-0-30
```

Comment:
I believe this needs to have the `.md` too, right?

Reply:
Most of the others don't...

Reply:
nvm then :)